numpy/_core/numeric.py

```diff
@@ -969,9 +969,11 @@ def tensordot(a, b, axes=2):
     * ``axes = 1`` : tensor dot product :math:`a\\cdot b`
     * ``axes = 2`` : (default) tensor double contraction :math:`a:b`
 
-    When `axes` is integer_like, the sequence for evaluation will be: first
-    the -Nth axis in `a` and 0th axis in `b`, and the -1th axis in `a` and
-    Nth axis in `b` last.
+    When `axes` is integer_like and if `axes` is a positive integer, 
+    the sequence for evaluation (multiply each element and then sum) will 
+    first start with the Nth position from the last (axis = -N) in `a` and 
+    the first position (axis = 0) in `b` and continue until the last position
+    (axis = -1) in `a` and Nth position (axis = N-1) in `b`.
```

Comment:
I think (multiply each element and then sum) makes clear what steps it is talking about though not necessary as mentioned in function summary.  I agree we can use "operation steps" rather than evaluation.